### PR TITLE
Initial assimilation into Rally.

### PR DESCRIPTION
**Pull request by jeffmay**
@danstaley @macmacbr @audax-shreyaspurohit @chrissalij-r @mingy711

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
+organizationName := "Rally Health"
 
-version := "0.1.0"
 scalaVersion := "2.11.7"
 
 licenses += ("Apache-2.0", url("http://opensource.org/licenses/apache-2.0"))
 
-// Publish to https://bintray.com/jeffmay/maven
-bintraySettings
-bintrayPublishSettings
```
Comment (chrissalij-r) [inline build.sbt]: Is the jeffmay bintray resolver below still needed?

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-  )(Resolver.ivyStylePatterns)
-)
+resolvers += Resolver.url(
+  "Rally Plugin Releases", 
+  url("https://artifacts.werally.in/artifactory/ivy-plugins-release")
+)(Resolver.ivyStylePatterns)
 
-// Allows publishing to bintray
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.2.1")
+addSbtPlugin("com.rallyhealth" %% "rally-sbt-plugin" % "0.4.0")
```
Comment (chrissalij-r) [inline project/plugins.sbt]: :)

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "latest.integration")
```
Comment (chrissalij-r) [inline project/plugins.sbt]: This needs to be above the sbt-plugin line I believe https://github.com/AudaxHealthInc/sortinghat/blob/master/project/plugins.sbt#L6

Comment (jeffmay) [inline project/plugins.sbt]: I wanted to test if this was still the case.

**Comment by jeffmay** [comment]
retest this please

File: build.sbt (hunk 1 above)
Comment (jeffmay) [inline build.sbt]: No, I'll remove it.

Comment (jeffmay) [inline build.sbt]: DONE

File: project/plugins.sbt [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
+resolvers += Resolver.url(
+  "Rally Plugin Releases", 
+  url("https://artifacts.werally.in/artifactory/ivy-plugins-release")
+)(Resolver.ivyStylePatterns)
 
-// Allows publishing to bintray
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.2.1")
+// Add Rally plugins
+addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "latest.integration")
+addSbtPlugin("com.rallyhealth" %% "rally-sbt-plugin" % "0.4.0")
```
Comment (jeffmay) [inline project/plugins.sbt]: I switched the order back. I don't think it matters anymore, but it's more consistent with other projects this way.

Comment (chrissalij-r) [inline project/plugins.sbt]: It does matter, but only when you're resolving new artifacts and publishing them. It won't trigger on compile or anything like that

**Comment by chrissalij-r** [comment]
LGTM

**Comment by audax-shreyaspurohit** [comment]
lgtm

**Comment by jeffmay** [comment]
Released as v0.1.0: https://rally-jenkins.werally.in/view/By%20Category/view/Release/job/Release_neo4j-scala-client/1/

**Pull request merged**